tools/genjava.js

```diff
@@ -15,123 +15,123 @@
  * limitations under the License.
  */
 
-// enable FOAM java support.
-global.FOAM_FLAGS = { 'java': true, 'debug': true };
+ // enable FOAM java support.
```

Comment:
My editor was being very weird. I just reverted to the last commit. Not sure where that white space came from